### PR TITLE
[website] Add CPU quantization tutorial

### PR DESCRIPTION
**Pull request by bgawrych**
## Description ##
Preview: https://hackmd.io/@bgaw/rJDp_hFcF

### CODE REVIEW

**Comment by bgawrych** [comment]
@mxnet-bot run ci [windows-gpu]

**Comment by bgawrych** [comment]
@mxnet-bot run ci [windows-gpu]

File: docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization.md [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+top1, top5 = test_accuracy(qnet, val_data)
+print('INT8Entropy Top1 Accuracy: {} Top5 Accuracy: {}'.format(top1, top5))
+```
+
+#### Output:
+> FP32 Top1 Accuracy: 0.76364 Top5 Accuracy: 0.93094
+> INT8Naive Top1 Accuracy: 0.76028 Top5 Accuracy: 0.92796
+> INT8Entropy Top1 Accuracy: 0.76404 Top5 Accuracy: 0.93042
+
+With quantized model there is tiny accuracy drop, however this is cost of great performance optimization and memory footprint. The difference between calibration method is dependent on the model itself, used activation layers and the size of calibration data.
```
Comment (bartekkuncer) [inline docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization.md:228]:
```suggestion
With quantized model there is tiny accuracy drop, however this is the cost of great performance optimization and memory footprint reduction. The difference between calibration methods is dependent on the model itself, used activation layers and the size of calibration data.
```

File: docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization.md [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+
+### Custom layer collectors and calibrating the model
+In MXNet 2.0 new interface for creating custom calibration collector has been added. Main goal of this interface is to give the user as much flexibility as possible in almost every step of quantization. Creating own layer collector is pretty easy, however computing effective min/max values can be not a trivial task. 
+
+Layer collectors are responsible for collecting statistics of each node in the graph — it means that the input/output data of every operator executed can be observed. Collector utilize the register_op_hook method of HybridBlock class.
```
Comment (bartekkuncer) [inline docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization.md:233]:
```suggestion
Layer collectors are responsible for collecting statistics of each node in the graph — it means that the input/output data of every operator executed can be observed. Collector utilizes the register_op_hook method of HybridBlock class.
```

File: docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization.md [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+Performance results of CV models. Chart presents three different runs: base float32 model without optimizations, fused float32 model with optimizations and quantized model.
+![performance](https://github.com/dmlc/web-data/blob/main/mxnet/tutorials/onednn/quantization_2_0/speedup.png?raw=true)
+###### Relative Inference Performance (img/sec) for Batch Size 128
+
+### Accuracy
+Accuracy results of CV models. Chart presents three different runs: base float32 model without optimizations, fused float32 model with optimizations and quantized model.
+![accuracy](https://github.com/dmlc/web-data/blob/main/mxnet/tutorials/onednn/quantization_2_0/accuracy.png?raw=true)
+###### ImageNet(ILSVRC2012) TOP1 validation accuracy
+
+##Notes
```
Comment (bartekkuncer) [inline docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization.md:302]:
```suggestion
## Notes
```

File: docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization.md [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+*optimize_for* function is available also as Symbol class method. Example call to this API is shown below. Notice that Symbol’s *optimize_for* method is not done in-place, so assigning it to a new variable is required:
+
+```
+optimized_symbol = sym.optimize_for(backend='ONEDNN')
+```
+
+For the above model definition in a naive benchmark with artificial data, we can gain up to 1.75x speedup without any accuracy loss on our testing machine with Intel(R) Core(TM) i9-9940X.
+
+
+## Quantization
```
Comment (bartekkuncer) [inline docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization.md:104]: Inconsistent new line after ## and ### - seems to be not too important as in HackMD preview it does look identical with or without it.

**Review by anko-intel — APPROVED** [review]

File: docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization.md (hunk 4 above)
Comment (bgawrych) [reply docs/python_docs/python/tutorials/performance/backend/dnnl/dnnl_quantization.md:104]: Done

**Review by bartekkuncer — APPROVED** [review]
LGTM

**Comment by bgawrych** [comment]
@mxnet-bot run ci [centos-gpu, windows-gpu]

**Comment by bgawrych** [comment]
@mxnet-bot run ci [windows-gpu]

**Comment by bgawrych** [comment]
@mxnet-bot run ci [windows-gpu]

**Pull request merged**